mitosheet/mitosheet/code_chunks/step_performers/column_steps/split_text_to_columns_code_chunk.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+
+from typing import List
+from mitosheet.code_chunks.code_chunk import CodeChunk
+from mitosheet.sheet_functions.types.utils import is_datetime_dtype, is_string_dtype
+from mitosheet.transpiler.transpile_utils import column_header_list_to_transpiled_code, column_header_to_transpiled_code
+
+
+class SplitTextToColumnsCodeChunk(CodeChunk):
+
+    def get_display_name(self) -> str:
+        return 'Split text to columns'
+    
+    def get_description_comment(self) -> str:
+        sheet_index = self.get_param('sheet_index')
```

Comment:
Everywhere you access params (here and elsewhere), might as well add types to them? If there other steps you see not doing this, go ahead and add them :-)